packages/recommend/src/types/TrendingFacetsQuery.ts

```diff
@@ -1,6 +1,21 @@
-import { TrendingQuery } from './TrendingQuery';
+export type TrendingFacetsQuery = {
+  /**
+   * The name of the target index.
+   */
+  readonly indexName: string;
 
-export type TrendingFacetsQuery = Omit<
-  TrendingQuery,
-  'model' | 'facetValue' | 'fallbackParameters' | 'queryParameters'
->;
+  /**
+   * Threshold for the recommendations confidence score (between 0 and 100). Only recommendations with a greater score are returned.
+   */
+  readonly threshold?: number;
+
+  /**
+   * How many recommendations to retrieve.
+   */
+  readonly maxRecommendations?: number;
+
+  /**
+   * Used for trending model
```

Comment:
This doesn't describe what this param is? (Same for `facetName` and `facetValue` for trending items.)